x-pack/README.md

```diff
@@ -100,6 +100,16 @@ We also have SAML API integration tests which set up Elasticsearch and Kibana wi
 node scripts/functional_tests --config test/saml_api_integration/config
 ```
 
+#### Running and building Jest integration tests
+
+Jest integration tests can be used to test behavior with Elasticsearch and the Kibana server.
+
+```sh
+node scripts/jest_integration
+```
+
+An example test exists at [test_utils/jest/integration_tests/example_integration.test.ts](test_utils/jest/integration_tests/example_integration.test.ts)
+
```

Comment:
I just saw this was merged. Nice! I've been looking forward this.

Are there any examples of how to create a jest integration test for X-Pack?

Reply:
nm x-pack/test_utils/jest/integration_tests/example_integration.test.ts